app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,12 @@
     <%= javascript_importmap_tags %>
   </head>
 
+  <h1>
+    <%= link_to "Home",jobs_path, method: :get, class:"home_link"%>
```

Comment:
I think here a better option would be `root_path`. 

I say this because it's possible your home page will change, and using `root_path` instead will make it future proof - your home page will always be what you've defined as your root path

Reply:
ah this is a smart idea, thanks!